docs/apps/zonation.md

```diff
@@ -17,53 +17,50 @@ __Zonation__ is available in Puhti with following versions:
 
 ## Usage
 
-Zonation is available in the __zonation__ module and the command line interface can be loaded with
-
-`module load zonation`
-
-After loading the Zonation module you can run zonation commands in the following way:
+Zonation is available in the __zonation__ module:
 
 ```
+module load zonation
 z5 <command arguments>
 ```
+Zonation can be used in Puhti with commandline or graphical interface, as interactive job or with batch system. In any case reserve suitable amount of computing resources: cores and memory. Zonation 5 runs faster, if it can use several cores. In Puhti, it can use one node, which is max 40 cores.  
```

Comment:
```suggestion
Zonation can be used in Puhti with command-line or graphical interface, as interactive job or with batch system. In any case reserve suitable amount of computing resources: cores and memory. Zonation 5 runs faster, if it can use several cores. In Puhti, it can use one node, which is max 40 cores.  
```